standard_logger.py

```diff
@@ -0,0 +1,19 @@
+import logging
+
+
+def login():
```

Comment:
We can call this method whenever we want to log.
The output message i.e.. log info is a random message as of now